core/schema/src/main/resources/org/hawkular/schema/bootstrap.groovy

```diff
@@ -48,8 +48,14 @@ if (reset) {
   executeCQL("DROP KEYSPACE IF EXISTS $keyspace", 20000)
 }
 
-executeCQL("CREATE KEYSPACE IF NOT EXISTS $keyspace WITH replication = {'class': 'SimpleStrategy', " +
-    "'replication_factor': $replicationFactor}")
+def keyspaceCreate = "CREATE KEYSPACE IF NOT EXISTS $keyspace WITH replication = {'class': 'SimpleStrategy', " +
+    "'replication_factor': $replicationFactor}"
+
+if (keyspace.equals("hawkulartest") || keyspace.equals("hawkular_metrics_rest_tests")) {
```

Comment:
In theory I also prefer making it configurable. In practice, I cannot recall when I have actually used different keyspaces for tests.